<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAdd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>82</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>查找数据:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonFind">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDelete">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSure">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>91</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>65</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>数据表名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>12</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUserName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassWord">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDataBaseName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTableName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>171</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
